src/components/plots/helpers/changeEmbeddingAxesIfNecessary.js

```diff
@@ -0,0 +1,15 @@
+const changeEmbeddingAxesIfNecessary = (config, method, onUpdate) => {
+  if (!config || !method) return;
+  const { defaultValues: axesDefaultValues, xAxisText, yAxisText } = config?.axes;
+
+  if (axesDefaultValues?.length) {
+    const methodUppercase = method[0].toUpperCase() + method.slice(1);
```

Comment:
can you give me an example of a value for `methodUppercase`? Also an example for values for `method`?

Reply:
method can be tsne or umap, so methodUppercase - Tsne, Umap

Reply:
so why does it have to be a list?